extensions/bitwarden/.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode
-raycast-env.d.ts
\ No newline at end of file
+raycast-env.d.ts
+node_modules
```

Comment:
Added `node_modules` here because its absence showed all dependencies in source control as new files upon running `npm run i`